src/api/flaskr/dao/__init__.py

```
@@ -29,9 +32,43 @@ def init_db(app: Flask):
+            ):
+                # Format the SQL statement
+                formatted_sql = sqlparse.format(
+                    statement, reindent=True, keyword_case="upper", strip_comments=True
+                )
+
+                # If there are parameters, try formatting
+                if parameters:
+                    try:
+                        # Try to format the parameters into the SQL statement
+                        raw_sql = formatted_sql % parameters
+                    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)